packages/ses-ava/package.json

```diff
@@ -22,7 +22,8 @@
   "exports": {
     ".": "./index.js",
     "./exported.js": "./exported.js",
-    "./package.json": "./package.json"
+    "./package.json": "./package.json",
+    "./prepare-endo.js": "./prepare-endo.js"
```

Comment:
✅ Got it in one.

Nit: we put `package.json` at the end of the list in all other packages.

Reply:
Done